core/services/vrf/listener_v2.go

```diff
@@ -448,15 +461,13 @@ func MaybeSubtractReservedLink(l logger.Logger, q pg.Q, startBalance *big.Int, c
 				   AND state IN ('unconfirmed', 'unstarted', 'in_progress')
 				   GROUP BY meta->>'SubId'`, chainID, subID)
 	if err != nil && !errors.Is(err, sql.ErrNoRows) {
-		l.Errorw("Could not get reserved link", "err", err)
-		return nil, err
+		return nil, fmt.Errorf("getting reserved LINK: %w", err)
```

Comment:
`fmt.Errorf` vs. `errors.Wrap` - which one should we use here? I don't have a particular preference for either, but I know we do use `errors.Wrap` alot.

Reply:
Ah yeah in this case it doesn't matter. Switched to errors.Wrap since there's no other fields we need to embed